thetis/options.py

```diff
@@ -498,13 +510,14 @@ class ModelOptions2d(CommonModelOptions):
     """Options for 2D depth-averaged shallow water model"""
     name = 'Depth-averaged 2D model'
     solve_tracer = Bool(False, help='Solve tracer transport').tag(config=True)
+    use_tracer_conservative_form = Bool(False, help='Solve 2D tracer transport in the conservative form').tag(config=True)
     use_wetting_and_drying = Bool(
         False, help=r"""bool: Turn on wetting and drying
 
         Uses the wetting and drying scheme from Karna et al (2011).
         If ``True``, one should also set :attr:`wetting_and_drying_alpha` to control the bathymetry displacement.
         """).tag(config=True)
-    wetting_and_drying_alpha = FiredrakeConstantTraitlet(
+    wetting_and_drying_alpha = FiredrakeScalarExpression(
         Constant(0.5), help=r"""
```

Comment:
I guess this change is part of a different PR (#186)?

Reply:
Sorry I forgot I'd merged that branch. I don't actually need it so I've changed it in the branch

Reply:
Instead of reverting the changes by hand it would be better to clean up the commit history. You'll need to rebase this branch, so you'll hit this issue again at least once again.